pages/settings/settings.js

```diff
@@ -11,8 +11,12 @@ var userAgentCheckbox = document.getElementById('checkbox-user-agent')
 var userAgentInput = document.getElementById('input-user-agent')
 
 function showRestartRequiredBanner () {
-  banner.hidden = false
+  banner.hidden = false;
```

Comment:
There shouldn't be a semicolon here right?

Reply:
I mean I just did that because I like semi-colons